setup.py

```diff
@@ -9,143 +9,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import annotations
-
-import glob
 import os
-import re
-import sys
 import warnings
 
-import pkg_resources
 from setuptools import find_packages, setup
 
 import versioneer
 
-# TODO: debug mode -g -O0, compile test cases
-
-RUN_BUILD = os.getenv("BUILD_MONAI", "0") == "1"
-FORCE_CUDA = os.getenv("FORCE_CUDA", "0") == "1"  # flag ignored if BUILD_MONAI is False
-
-BUILD_CPP = BUILD_CUDA = False
-TORCH_VERSION = 0
-try:
-    import torch
-
-    print(f"setup.py with torch {torch.__version__}")
-    from torch.utils.cpp_extension import BuildExtension, CppExtension
-
-    BUILD_CPP = True
-    from torch.utils.cpp_extension import CUDA_HOME, CUDAExtension
-
-    BUILD_CUDA = FORCE_CUDA or (torch.cuda.is_available() and (CUDA_HOME is not None))
```

Comment:
Removing this variable significantly changes the logic of what's going on here. If CUDA isn't present we don't want to try building against it, this is why some of the CICD tests fail. Below we've used this variable to choose whether to add the CUDAExtension object or not.